wg-releases/triaging-issues.md

```diff
@@ -63,6 +72,64 @@ Some reporters will provide a small snippet without context, and if you can, we
 
 > Thanks in advance! Your help is appreciated.
 
+## Common Responses
+
+This section outlines several common scenarios, and the responses and labels we recommend using for issue triage. It also tries to address several commonly asked questions for those new to the issue triage process.
+
+_NB: The reponses below are meant to be a guide for someone new to triaging issues in Electron, not requirements. If you find an issue while triaging that you think needs a different response than is recommended here, use your own judgement._
+
+#### Low Quality/Spam
+
+Issue did not fill out the issue template:
+  * Reply with [`issue-template` response](https://github.com/electron/governance/blob/main/wg-releases/issue-playbook/fill-template.md)
+  * Add `blocked/needs-info` label
+
+Issue is asking about a third-party dependency, or app-specific JavaScript code
+  * Reply with `send-to-discord` response
+  * Close the issue
+
+Issue is transparently spam
+  * Close the issue (no reply needed)
+
+#### Blocked/Needs Info
+
+Issue does not contain adequate information (i.e., no expected behavior or is on an older version of Electron):
```

Comment:
```suggestion
Issue does not contain adequate information (i.e., no expected behavior or is on an unsupported version of Electron):
```
There can be reports about an older version of electron that is still supported, right?

Reply:
Yeah you're right, this language is more precise - I'll amend this, thanks! 🙂